src/parser/user-extractor-transformer.ts

```diff
@@ -0,0 +1,68 @@
+import configuration from "../configuration/config-reader";
+import { GetActivity } from "../get-activity";
+import { GitHubIssue, GitHubIssueComment } from "../github-types";
+import { Result, Transformer } from "./processor";
+
+/**
+ * Creates entries for each bounty hunter with its associated comments.
+ */
+export class UserExtractorTransformer implements Transformer {
+  private readonly _configuration = configuration["user-extractor"];
+
+  get enabled(): boolean {
+    return true;
+  }
+
+  /**
+   * Checks if the comment is made by a human user, and if it not a command.
+   * @param comment
+   */
+  _checkEntryValidity(comment: GitHubIssueComment) {
+    return !!comment.user && !!comment.body && comment.user.type === "User";
+  }
+
+  _extractBountyPrice(issue: GitHubIssue) {
+    if (this._configuration["redeem-bounty"] === false) {
+      return 0;
+    }
+    const sortedPriceLabels = issue.labels
+      .reduce((acc, label) => {
+        const labelName = typeof label === "string" ? label : label.name;
+        if (labelName?.startsWith("Price: ")) {
+          const price = parseFloat(labelName.replace("Price: ", ""));
+          if (!isNaN(price)) {
+            acc.push(price);
+          }
+        }
+        return acc;
+      }, [] as Array<number>)
+      .sort((a, b) => {
+        return a - b;
+      });
+    if (!sortedPriceLabels.length) {
+      console.warn("There are no price labels in this repository.");
+      return 0;
+    }
+    return sortedPriceLabels[0];
+  }
+
+  transform(data: Readonly<GetActivity>, result: Result): Result {
+    if (data.comments) {
+      for (const comment of data.comments as GitHubIssueComment[]) {
+        if (comment.user && comment.body && this._checkEntryValidity(comment)) {
+          const bounty =
```

Comment:
I actually deliberately changed the "bounty" terminology to "task" (or "tasks") in order to appeal to a wider demographic. Bounties seem to be very crypto audience focused, but I want our system to add value to non crypto companies as well. 

This makes sense to reflect this decision from our marketing materials down to our code so that there is no confusion within our organization. 

Also I generally refer to them as tasks only after they are funded. Otherwise I just refer to them as GitHub issues (or issues.)

And if there are any problems in the codebase etc I am deliberate to not use the term "issue" as it can be interpreted as a reference to a GitHub issue. Instead, I call them another synonym, like "problem"

Reply:
Sounds good to me! On the other hand `bounty` refers to a task with a prize, bounty hunting is also common for bug hunts, hackathons, and a common word amongst developers that are not doing crypto.

Reply:
Maybe that's true in your and my experience. But I was told otherwise from some other developers. Perhaps outside of gaming and crypto it's not a widely used term?

Reply:
Could be a reason for sure, will rename to `task`